extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/MongoClientProcessor.java

```diff
@@ -75,9 +78,14 @@ UnremovableBeanBuildItem markBeansAsUnremovable() {
     void configureRuntimeProperties(MongoClientRecorder recorder,
             CodecProviderBuildItem codecProvider,
             BsonDiscriminatorBuildItem bsonDiscriminator,
-            MongodbConfig config) {
+            MongodbConfig config, List<MongoConnectionPoolListenerBuildItem> connectionPoolListenerProvider) {
```

Comment:
Can you move the connectionPoolListenerProvider on a new line, this will be easier to read